src/backend/server.js

```diff
@@ -3,14 +3,13 @@ const express = require('express');
 const app = express();
 const port = 5000;
 
-app.get("/api/menuitems", (req, res) => {
-    var menuItems = [
-        {id: 1, description: "Chocolate Cheesecake"},
-        {id: 2, description: "Strawberry Cheesecake"},
-        {id: 3, description: "Vanilla Cheesecake"},
-    ];
-    res.json(menuItems);
-})
+app.get('/api/menuitems', (req, res) => {
```

Comment:
Hi @nilan01,

Can you please explain why this is being edited in this PR?

Reply:
Hi @chrispinkney,

Good question. This is Prettier correcting code formatting upon saving changes.